### PR TITLE
Study version of LLNL's CoMD

### PR DESCRIPTION
**Pull request by benharsh**
This implementation of CoMD in Chapel will serve as a point of comparison for elegance and performance when we write a more idiomatic version in Chapel.

TODO:
- [x] 500 correctness trials
- [x] verify perf/graph files work

### CODE REVIEW

**Comment by benharsh** [comment]
@ben-albrecht : would you be available to review?

File: test/studies/comd/llnl/LICENSE [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/******************************************************************************
+
+CoMD-Chapel v1.0
+
+Copyright (c) 2016, Lawrence Livermore National Security, LLC.
```
Comment (ben-albrecht) [inline test/studies/comd/llnl/LICENSE]: Does this need to be updated?

Comment (benharsh) [inline test/studies/comd/llnl/LICENSE]: Perhaps, but I think that's on LLNL. I just copied this license from their repo.

Comment (benharsh) [inline test/studies/comd/llnl/LICENSE]: @tmacd9  : does that sound right to you?

**Pull request merged**